src/main/java/net/sf/jabref/importer/fetcher/OAI2Fetcher.java

```diff
@@ -220,11 +220,11 @@ public BibEntry importOai2Entry(String key) {
             }
             return be;
         } catch (IOException e) {
-            status.showMessage(Localization.lang("An Exception occurred while accessing '%0'", url) + "\n\n" + e,
+            status.showMessage(Localization.lang("An exception occurred while accessing '%0'", url) + "\n\n" + e,
                     getTitle(), JOptionPane.ERROR_MESSAGE);
         } catch (SAXException e) {
             status.showMessage(
-                    Localization.lang("An SAXException occurred while parsing '%0':", url) + "\n\n" + e.getMessage(),
+                    Localization.lang("An SAX exception occurred while parsing '%0':", url) + "\n\n" + e.getMessage(),
```

Comment:
I'm not a native speaker but typically I'd say this has no vowel or vowel sound so it's an A.